### PR TITLE
docs: add links back to cheat sheet

### PR DESCRIPTION
**Pull request by huynhicode**
# Pull Request

## 📖 Description

Linking back to the Cheat Sheet from the Docs.

### 🎫 Issues

None

## 👩‍💻 Reviewer Notes

Please review the verbiage and corresponding links.

## 📑 Test Plan

None

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by awentzel — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

**Review by janechu — APPROVED** [review]

File: packages/web-components/fast-element/README.md [hunk 1]
```diff
@@ -55,4 +55,8 @@ The markup above always references the latest release. When deploying to product
 
 :::note
 For simplicity, examples throughout the documentation will assume the library has been installed from NPM, but you can always replace the import location with the CDN URL.
+:::
+
+:::tip
+Looking for a quick guide on building components?  Check out [our Sheet Cheat](https://www.fast.design/docs/resources/cheat-sheet#building-components).
```
Comment (EisenbergEffect) [inline packages/web-components/fast-element/README.md:61]:
```suggestion
Looking for a quick guide on building components?  Check out [our Cheat Sheet](https://www.fast.design/docs/resources/cheat-sheet#building-components).
```

File: packages/web-components/fast-element/docs/guide/declaring-templates.md [hunk 2]
```diff
@@ -223,3 +223,7 @@ In the future we're planning new optimizations that will enable us to safely det
 In most cases, the template that `FASTElement` renders is determined by the `template` property of the Custom Element's configuration. However, you can also implement a method on your Custom Element class named `resolveTemplate()` that returns a template instance. If this method is present, it will be called during `connectedCallback` to obtain the template to use. This allows the element author to dynamically select completely different templates based on the state of the element at the time of connection.
 
 In addition to dynamic template selection during the `connectedCallback`, the `$fastController` property of `FASTElement` enables dynamically changing the template at any time through setting the controller's `template` property to any valid template.
+
+:::tip
+Check out [our Sheet Cheat](https://www.fast.design/docs/resources/cheat-sheet#bindings) for a quick guide on bindings.
```
Comment (EisenbergEffect) [inline packages/web-components/fast-element/docs/guide/declaring-templates.md:228]:
```suggestion
Check out [our Cheat Sheet](https://www.fast.design/docs/resources/cheat-sheet#bindings) for a quick guide on bindings.
```

File: packages/web-components/fast-element/docs/guide/next-steps.md [hunk 3]
```diff
@@ -9,4 +9,6 @@ We've seen how to use `FASTElement` to declaratively build Web Components. In ad
 
 Now that you're familiar with the robust and powerful features of `FASTElement`, you're ready to build your own components and apps. But you don't have to start from scratch there either! If you haven't already explored them, check out our [FAST Components](../components/getting-started), which provide all the basic UI building-blocks you'd expect in a modern component library. You can also leverage the same adaptive design system that our own components use to enable robust theming throughout all you create. Read more on that in [Styling Components](../design/introduction). Finally, you'll want to take advantage of a modern toolset by installing [a powerful editor and plugins](../tools/vscode).
 
+For a quick guide, check out [our Cheat Sheet](https://www.fast.design/docs/resources/cheat-sheet).
```
Comment (EisenbergEffect) [inline packages/web-components/fast-element/docs/guide/next-steps.md:12]:
```suggestion
For a quick reference on what you've learned, check out [our Cheat Sheet](https://www.fast.design/docs/resources/cheat-sheet).
```

File: sites/website/src/docs/introduction.md [hunk 4]
```diff
@@ -41,6 +41,8 @@ We hope you're excited by the possibilities that FAST presents. But, you may be
   * [Jump to the fast-element docs.](/docs/fast-element/getting-started)
 * "I need to integrate FAST with another framework or build system."
   * [Jump to the integration docs.](/docs/integrations/introduction)
+* "I want to look at a quick guide."
```
Comment (EisenbergEffect) [inline sites/website/src/docs/introduction.md:44]:
```suggestion
* "I want to look at a quick reference."
```

**Review by EisenbergEffect — CHANGES_REQUESTED** [review]

**Comment by huynhicode** [comment]
@EisenbergEffect Thank you for those catches and verbiage update suggestions!

**Review by EisenbergEffect — APPROVED** [review]

**Pull request merged**